artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnection.java

```diff
@@ -46,40 +43,37 @@
 
 public class NettyConnection implements Connection {
 
-   // Constants -----------------------------------------------------
-   private static final int BATCHING_BUFFER_SIZE = 8192;
-
-   // Attributes ----------------------------------------------------
+   private static final int DEFAULT_MTU_BYTES = Integer.getInteger("io.netty.mtu", 1460);
+   //backpressure on unbatched writes is enabled by default
```

Comment:
Any form of configuration here would have to be done through connection properties.. on the case through the URI that will specify the connector...


I'm not saying I agree with these.. more to follow